uiServices/src/configServices.go

```diff
@@ -59,11 +51,12 @@ func ConfigCtx(next http.Handler) http.Handler {
 }
 
 func postConfigs(rw http.ResponseWriter, req *http.Request) {
-	configPathDir := req.Header.Get("configPathDir")
+	rw.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
i think we could have a return method at the end of each handler that adds this header, the body, and the status code to the request.